data/table.txt

```diff
@@ -14678,7 +14931,8 @@
 沆 沆 1 1 0 0 1 0 0 0 0 eyhn eyhn NA 22181
 沇 沇 1 1 0 0 1 0 0 0 0 eihu eihu NA 13929
 沈 沈 1 1 0 0 1 0 0 0 0 elbu elbu NA 22198
-沉 沈 1 1 0 0 1 0 0 0 0 ebhn,ebhu ebhn,ebhu,xebhu NA 22197
+沉 沈 1 1 0 0 1 0 0 0 0 ebhn ebhn NA 22197
+沉 沈 1 1 0 0 1 0 0 0 0 ebhu ebhu,xebhu NA 0
```

Comment:
This was identified by @dollars0427 as a problem.

---

In Cangjie 3, for 「水*弓」 (e*n), we have :

* 汀沉汽沆沸洌涔涮湔 with libcangjie
* 汀　汽沆沸洌涔涮湔 with Hime

So we can clearly see that we added 沉 in there, moving everything else back.

沉 has 2 codes, `ebhn` and `ebhu`.

It seems that the primary code on Windows for 沉 should have been `ebhu`, with `ebhn` being the secondary one? It would be nice if someone could confirm on Windows.

Reply:
I'm told that on Windows, `ebhu` is indeed 沉, and that `ebhn` has no character associated to it.

So for 沉, the primary code should be `ebhu`, and we should either remove `ebhn` or at the very least make it the secondary character.

Reply:
Talking with @yookoala, we decided to keep `ebhn` but make `ebhu` the primary code.

This is now fixed in the latest push to this branch.